### DISCUSSION TITLE
Error while creating Ui

### DISCUSSION THREAD

**Discussion by rohandixitdelhi95** [Sunbird-cQube/community#833] [General]
We are creating a new Ui for pat and for that we have created four filters and on tab with one report i.e(drilldown map). 
And a filter checkbox for not applying the filter.
In console i am getting this error,could you please guide us how to resolve this error.

![8e8083a5-36f5-4cdc-969b-560229b87c72](https://github.com/Sunbird-cQube/community/assets/140483814/c4f62031-8fb6-4feb-aead-0fa9c414bcce)

**Comment by rohandixitdelhi95** [comment]
<img width="772" alt="image" src="https://github.com/Sunbird-cQube/community/assets/140483814/b10257dd-48b7-43ee-aaf1-6852e90569e8">

**Reply by pandutibil** [reply]
Please join here [https://meet.google.com/ywg-teaq-qgu](https://meet.google.com/ywg-teaq-qgu)

**Comment by RNarmatha** [comment]
Hi @rohandixitdelhi95 - Could you please confirm the issue got resolved ? If not any support is needed to fix it ?
